packages/yourloops/pages/hcp/team-leave-dialog.tsx

```diff
@@ -0,0 +1,267 @@
+        <strong>{t("team-leave-dialog-only-admin-title")}</strong>
+        <br />
+        <span id="team-leave-dialog-title-team-name">{teamName}</span>
+      </DialogTitle>
+    );
+  }
+
+  return (
+    <DialogTitle id="team-leave-dialog-title">
+      <strong>{t("team-leave-dialog-title")}</strong>
+      <br />
+      <span id="team-leave-dialog-title-team-name">{teamName}</span>
+    </DialogTitle>
+  );
+}
+
+function LeaveTeamDialogContent(props: LeaveTeamDialogElementsProps): JSX.Element | null {
+  const { team, teamName, onlyMember, userIsTheOnlyAdministrator } = props;
+
+  if (team === null) {
+    return null;
+  }
+
+  if (onlyMember) {
+    return (
+      <DialogContent>
+        <DialogContentText id="team-leave-dialog-question">
+          <Trans i18nKey="team-leave-dialog-and-del-question" t={t} components={{ strong: <strong /> }} parent={React.Fragment}>
+            Since you are the only &quot;member&quot; in this team, {{ teamName }} will be <strong>permanently deleted</strong> if you leave it.
+          </Trans>
+        </DialogContentText>
+        <DialogContentText id="team-leave-dialog-consequences">
+          {t("team-leave-dialog-and-del-consequences")}
+        </DialogContentText>
+      </DialogContent>
+    );
+  }
+
+  if (userIsTheOnlyAdministrator) {
+    return (
+      <DialogContent>
+        <DialogContentText id="team-leave-dialog-consequences">
+          <Trans i18nKey="team-leave-dialog-only-admin-consequences" t={t} components={{ strong: <strong /> }} parent={React.Fragment}>
+            Since you are the only team&apos;s administrator, you need to <strong>give administrator permission to another member</strong> before leaving.
+          </Trans>
+        </DialogContentText>
+      </DialogContent>
+    );
+  }
+
+  return (
+    <DialogContent>
+      <DialogContentText id="team-leave-dialog-question">
+        {t("team-leave-dialog-question")}
+      </DialogContentText>
+      <DialogContentText id="team-leave-dialog-consequences">
+        {t("team-leave-dialog-consequences")}
+      </DialogContentText>
+    </DialogContent>
+  );
+}
+
+function LeaveTeamDialogActions(props: LeaveTeamDialogElementsProps): JSX.Element | null {
+  const { team, onlyMember, userIsTheOnlyAdministrator, buttonsDisabled, handleClose, handleLeaveTeam } = props;
+  const classes = leaveTeamDialogClasses();
+
+  if (team === null) {
+    return null;
+  }
+
+  if (onlyMember) {
+    return (
+      <DialogActions>
+        <Button id="team-leave-dialog-button-cancel" onClick={handleClose} disabled={buttonsDisabled} className={classes.buttonCancel} color="secondary" variant="contained">
+          {t("Cancel")}
+        </Button>
+        <Button id="team-leave-dialog-button-leave" onClick={handleLeaveTeam} disabled={buttonsDisabled} color="secondary" className={classes.buttonLeaveAndDel} variant="contained">
+          {t("team-leave-dialog-button-leave-and-del")}
+        </Button>
+      </DialogActions>
```

Comment:
Same, we can probably avoid these duplications

Reply:
A PR to fix all theses little things on the team screen will follow.